### DISCUSSION TITLE
$listeners not working with laravel Echo

### DISCUSSION THREAD

**Discussion by bynicodevelop** [livewire/livewire#3566] [7. Bugs (archived)]
Hello,

I am your documentation on Livewire and Laravel Echo: https://laravel-livewire.com/docs/2.x/laravel-echo

I have a laravel echo configuration which works very well in Javascript and I receive my events well (in a js script in a livewire component).

Like this : 
```javascript
document.addEventListener('DOMContentLoaded', function() {
                console.log("loaded");
                window.Echo.private('post-created-event')
                    .listen('PostCreatedEvent', (e) => {
                        console.log(e);
                    });
            })
```

I am trying to delete this script to reproduce that which is written in the doc

like this : 

```php
public function getListeners()
    {
        Log::debug('echo-private:private-post-created-event,PostCreatedEvent');

        return [
            'echo-private:private-post-created-event,PostCreatedEvent' => 'notifyNewPostCreated',
            // 'echo-private:private-post-created-event,' . PostCreatedEvent::class => 'notifyNewPostCreated', <-- I tried it :(
        ];
    }

public function notifyNewPostCreated()
    {
        Log::debug("new event");
    }

```

unfortunately, the event is not launched.

Do you have an idea ?
Is a configuration problem on my part?

Or a bug already mentioned?

Thanks

**Comment by Loots-it** [comment]
I have the same issue. I'm using the latest version of livewire so that can't be the issue. Did you find a solution yet?
